.travis.yml

```diff
@@ -3,3 +3,4 @@ node_js:
   - iojs
   - 0.12
   - 0.10
+#after_success: cat coverage/lcov.info | node_modules/.bin/coveralls --verbose
```

Comment:
After you log in to https://coveralls.io, removing the pound sign should be enough to generate coverage reports